config/routes/annotations.yaml

```diff
@@ -1,3 +1,5 @@
 controllers:
     resource: ../../src/Controller/
     type: annotation
+    options:
+        expose: true
```

Comment:
This doesn't seem to make any difference? Where does it break if you change this?

Reply:
Try running the fos:js-routing:dump/debug, it should be empty without that config

Reply:
I can't seem to make it fail?